website/content/docs/enterprise/federation.mdx

```diff
@@ -33,3 +33,20 @@ can make queries to the remote datacenter in service of both API and DNS
 requests for remote resources (in spite of the partially-connected nature of the
 topology as a whole). Consul datacenters can simultaneously participate in both
 network areas and the existing WAN pool, which eases migration.
+
+## Requirements
+
+- Consul Enterprise 0.8+
+- **Direct server connectivity:**
+  All Consul servers in the 2 Consul datacenters
+  you want to connect with a network area must have direct network connection. 
+  To ensure connectivity is direct, not funneled through mesh gateways,
+  verify that the
+  [`connect.enable_mesh_gateway_wan_federation`](/docs/agent/config/config-files#connect_enable_mesh_gateway_wan_federation)
+  option is set to `false` (default).
+- **Primary datacenter connectivity:**
+  All Consul datacenters must be connected
+  to the [primary_datacenter](/docs/agent/config/config-files#primary_datacenter)
+  through WAN gossip or a network area.
+  Network areas cannot be if two datacenters are connected only through a
+  [cluster peering](/docs/connect/cluster-peering) relationship.
```

Comment:
```suggestion
- Consul Enterprise 0.8+
- **Direct server connectivity between all Consul servers in each datacenter:**
  Consul servers in each datacenter you want to connect in the network area must have a direct network connection. 
  To ensure connectivity is direct and not funneled through mesh gateways,
  verify that the
  [`connect.enable_mesh_gateway_wan_federation`](/docs/agent/config/config-files#connect_enable_mesh_gateway_wan_federation)
  option is set to `false` (default).
- **Primary datacenter connectivity:**
  All Consul datacenters must be connected to the [primary_datacenter](/docs/agent/config/config-files#primary_datacenter)
  through WAN gossip or a network area. You cannot connect network areas if datacenters peered (refer to [Cluster Peering](/docs/connect/cluster-peering) for documentation on peering clusters).
```
Are you limited to federating two datacenters? Could you have more than one secondary DC?

Reply:
A single network area is always a relationship between 2 datacenters.

You can have as many secondary datacenters as you want with WAN federation. But a network area within a WAN federation connects a pair of datacenters (primary <-> secondary, or secondary <-> secondary).

Any thoughts on how to make this clearer in the text?